src/SourceBuild/content/repo-projects/arcade.proj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <!-- Use a prebuilt Arcade to build Arcade -->
     <UseBootstrapArcade>true</UseBootstrapArcade>
+
+    <!-- Remove after arcade bootstrap version is updated. -->
+    <BuildArgs>$(BuildArgs) /p:DotNetBuildSkipTests=true</BuildArgs>
```

Comment:
AFAIK, Arcade bootstrap is already recent enough for this change.

Reply:
If a bootstrap is needed, please open an issue and note this cleanup task.

Reply:
https://github.com/dotnet/installer/commit/530bfbd9d4b1d2232df9c1c6e45031256602f613 updated the version in V.D.xml to a new enough version. However, it didn't update Arcade.Sdk in global.json. So that's why. Fixing.